client/extensions/relatedArticles/src/EventsRelatedArticles/utils.ts

```diff
@@ -0,0 +1,196 @@
+import {IArticle} from 'superdesk-api';
+
+export const fakeEditor: any = {
```

Comment:
I'm assuming this is not needed for production, only in testing/development. If so, please remove this

Reply:
We do need it, because the items coming from Belga don't have an actual content profile, but the `search_provider_proxy` from our backend works with projections, and those are gotten through a content profile. In the future I guess we can rework this so we don't need to fake content profiles.